lpm/src/trie/prefix_map_impl.rs

```diff
@@ -132,6 +132,17 @@ where
             .get_lpm(&IpPrefixW(addr.into()))
             .map(|(p, v)| (&p.0, v))
     }
+
+    fn matching_entries<A>(&self, addr: A) -> impl Iterator<Item = (&P, &V)>
+    where
+        A: Into<Self::Prefix>,
+    {
+        self.0
```

Comment:
Is this a linear search over the entire lpm?  If so this is not acceptable, it will have terrible performance.

If this is a more clever walk of only the populated subtrees it could be ok for now, though we probably want a better data structure long term anyway.

Reply:
As discussed, it's using this method from the underlying structure:
https://docs.rs/prefix-trie/latest/prefix_trie/map/struct.PrefixMap.html#method.cover

It walks the subtrees so should be OK, at least until we find a better data structure.